pkg/configuration/new_asset_store.go

```diff
@@ -37,6 +37,12 @@ func NewAssetStoreAndCASFromConfiguration(configuration *pb.AssetCacheConfigurat
 		}
 		contentAddressableStorage = cas
 		assetStore = storage.NewActionCacheAssetStore(actionCache, contentAddressableStorage, maximumMessageSizeBytes)
+	case *pb.AssetCacheConfiguration_CasOnly:
```

Comment:
Should this also be a default case if the asset cache is not specified?

Reply:
The asset cache has to be specified in otherwise there won't be a specification of the CAS. There should be a default case which returns an error though which I will add.

Reply:
This sort of makes me feel like if the CAS is always required, it should be defined in the base configuration rather than inside an asset cache. Then asset cache definitions could pull in the CAS definition from the top level configuration.

Reply:
Yeah I did think about that, I'll have a play around and see if I can do that, I think the reason I didn't was to make best use of existing bb-storage functions to create the CAS and AC from the blobstore config.